### ISSUE TITLE
Certain libvirt commands fail on fedora-29 because of json parsing bug

### ISSUE THREAD

**Issue by Gundersanne** [cockpit-project/cockpit#10005]
https://bugzilla.redhat.com/show_bug.cgi?id=1614959 causes some check-machines tests to fail.

**Comment by cockpituous** [comment]
verify/fedora-29
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testBasic (check_machines.TestMachines)
#
[0906/171030.469561:ERROR:gpu_process_transport_factory.cc(1017)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:10120/devtools/browser/888c926d-a3e7-42a1-a178-bd0ff5bab1bf
[0906/171030.524693:ERROR:zygote_host_impl_linux.cc(267)] Failed to adjust OOM score of renderer with pid 103852: Permission denied (13)
[0906/171030.650502:ERROR:zygote_host_impl_linux.cc(267)] Failed to adjust OOM score of renderer with pid 103876: Permission denied (13)
> warning: Failed to query RHEL subscription status: 
error: Failed to suspend domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'
Wrote screenshot to TestMachines-testBasic-fedora-29-127.0.0.2-2401-FAIL.png
Wrote HTML dump to TestMachines-testBasic-fedora-29-127.0.0.2-2401-FAIL.html
Wrote JS log to TestMachines-testBasic-fedora-29-127.0.0.2-2401-FAIL.js.log
Journal extracted to TestMachines-testBasic-fedora-29-127.0.0.2-2401-FAIL.log
Traceback (most recent call last):
  File "/build/cockpit/bots/../test/verify/check-machines", line 275, in testBasic
    m.execute("virsh suspend subVmTest1")
  File "/build/cockpit/bots/machine/testvm.py", line 510, in execute
    raise subprocess.CalledProcessError(proc.returncode, command, output=output)
subprocess.CalledProcessError: Command 'virsh suspend subVmTest1' returned non-zero exit status 1.

not ok 50 testBasic (check_machines.TestMachines) # duration: 36s
```
First occurrence: 2018-09-06T17:10:44.205061 | revision 5e0693adaea8546859794993d9e0860fcf8fc67b, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-9916-20180906-163232-5e0693ad-verify-fedora-29/log.html)
Times recorded: 1
Latest occurrences:

- 2018-09-06T17:10:44.205061 | revision 5e0693adaea8546859794993d9e0860fcf8fc67b, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-9916-20180906-163232-5e0693ad-verify-fedora-29/log.html)

**Comment by cockpituous** [comment]
verify/fedora-29
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testDisks (check_machines.TestMachines)
#
[0906/171003.926949:ERROR:gpu_process_transport_factory.cc(1017)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:9618/devtools/browser/7ffdcd24-67bd-46dd-aee5-2b7f11fda909
[0906/171003.990871:ERROR:zygote_host_impl_linux.cc(267)] Failed to adjust OOM score of renderer with pid 103352: Permission denied (13)
[0906/171004.130171:ERROR:zygote_host_impl_linux.cc(267)] Failed to adjust OOM score of renderer with pid 103379: Permission denied (13)
> warning: Failed to query RHEL subscription status: 
Libvirt version does not support disk statistics
> warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

> info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
> warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

> info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
> warning: Checking for available updates failed: Error: cannot update repo 'fedora-modular': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]; Last error: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]
> warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

> info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
> warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

> info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
> warning: Checking for available updates failed: Error: cannot update repo 'fedora-modular': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]; Last error: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]
> warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

> info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
> warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

> info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
> warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

> info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
Wrote screenshot to TestMachines-testDisks-fedora-29-127.0.0.2-2601-FAIL.png
Wrote HTML dump to TestMachines-testDisks-fedora-29-127.0.0.2-2601-FAIL.html
Wrote JS log to TestMachines-testDisks-fedora-29-127.0.0.2-2601-FAIL.js.log
Journal extracted to TestMachines-testDisks-fedora-29-127.0.0.2-2601-FAIL.log
Traceback (most recent call last):
  File "/build/cockpit/bots/../test/verify/check-machines", line 366, in wait_for_disk_stats
    b.wait_present("#vm-{0}-disks-{1}-used".format(name, target)) # wait for disk statistics to show up
  File "/build/cockpit/test/common/testlib.py", line 287, in wait_present
    return self.wait_js_func('ph_is_present', selector)
  File "/build/cockpit/test/common/testlib.py", line 281, in wait_js_func
    self.wait_js_cond("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "/build/cockpit/test/common/testlib.py", line 278, in wait_js_cond
    self.raise_cdp_exception("timeout\nwait_js_cond", cond, result["exceptionDetails"], trailer)
  File "/build/cockpit/test/common/testlib.py", line 167, in raise_cdp_exception
    raise Error("%s(%s): %s" % (func, arg, msg))
testlib.Error: timeout
wait_js_cond(ph_is_present("#vm-subVmTest1-disks-hda-used")): {'exceptionId': 1, 'text': 'Uncaught (in promise) Error: condition did not become true', 'lineNumber': 194, 'columnNumber': 9, 'scriptId': '15', 'stackTrace': {'callFrames': [{'functionName': 'PhWaitCondTimeout', 'scriptId': '15', 'url': '', 'lineNumber': 193, 'columnNumber': 8}, {'functionName': 'window.setTimeout', 'scriptId': '15', 'url': '', 'lineNumber': 205, 'columnNumber': 23}]}, 'exception': {'type': 'object', 'subtype': 'error', 'className': 'PhWaitCondTimeout', 'description': 'PhWaitCondTimeout: condition did not become true\n    at window.setTimeout (<anonymous>:206:24)', 'objectId': '{"injectedScriptId":5,"id":1}'}}
cdp: {'source': 'network', 'level': 'error', 'text': 'Failed to load resource: the server responded with a status of 401 (Authentication failed)', 'timestamp': 1536253804451.04, 'url': 'http://127.0.0.2:9491/cockpit/login', 'networkRequestId': '103379.8'}
warning: Failed to query RHEL subscription status: 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/build/cockpit/bots/../test/verify/check-machines", line 462, in testDisks
    self.wait_for_disk_stats("subVmTest1", "hda")
  File "/build/cockpit/bots/../test/verify/check-machines", line 372, in wait_for_disk_stats
    b.wait_present("#vm-{0}-disksstats-unsupported".format(name))
  File "/build/cockpit/test/common/testlib.py", line 287, in wait_present
    return self.wait_js_func('ph_is_present', selector)
  File "/build/cockpit/test/common/testlib.py", line 281, in wait_js_func
    self.wait_js_cond("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "/build/cockpit/test/common/testlib.py", line 278, in wait_js_cond
    self.raise_cdp_exception("timeout\nwait_js_cond", cond, result["exceptionDetails"], trailer)
  File "/build/cockpit/test/common/testlib.py", line 167, in raise_cdp_exception
    raise Error("%s(%s): %s" % (func, arg, msg))
testlib.Error: timeout
wait_js_cond(ph_is_present("#vm-subVmTest1-disksstats-unsupported")): {'exceptionId': 3, 'text': 'Uncaught (in promise) Error: condition did not become true', 'lineNumber': 194, 'columnNumber': 9, 'scriptId': '15', 'stackTrace': {'callFrames': [{'functionName': 'PhWaitCondTimeout', 'scriptId': '15', 'url': '', 'lineNumber': 193, 'columnNumber': 8}, {'functionName': 'window.setTimeout', 'scriptId': '15', 'url': '', 'lineNumber': 205, 'columnNumber': 23}]}, 'exception': {'type': 'object', 'subtype': 'error', 'className': 'PhWaitCondTimeout', 'description': 'PhWaitCondTimeout: condition did not become true\n    at window.setTimeout (<anonymous>:206:24)', 'objectId': '{"injectedScriptId":5,"id":3}'}}
cdp: {'source': 'network', 'level': 'error', 'text': 'Failed to load resource: the server responded with a status of 401 (Authentication failed)', 'timestamp': 1536253804451.04, 'url': 'http://127.0.0.2:9491/cockpit/login', 'networkRequestId': '103379.8'}
warning: Failed to query RHEL subscription status: 
warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
warning: Checking for available updates failed: Error: cannot update repo 'fedora-modular': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]; Last error: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]
warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
warning: Checking for available updates failed: Error: cannot update repo 'fedora-modular': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]; Last error: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-modular-29&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]
warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"
warning: error: Failed to get memory statistics for domain subVmTest1
error: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'

info: The 'virsh' command failed, as expected: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"error: Failed to get memory statistics for domain subVmTest1\nerror: internal error: failed to parse JSON 1:63: too big integer near '18446744073709551615'"}", data: """"

not ok 53 testDisks (check_machines.TestMachines) # duration: 100s
```
First occurrence: 2018-09-06T17:11:27.394882 | revision 5e0693adaea8546859794993d9e0860fcf8fc67b, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-9916-20180906-163232-5e0693ad-verify-fedora-29/log.html)
Times recorded: 1
Latest occurrences:

- 2018-09-06T17:11:27.394882 | revision 5e0693adaea8546859794993d9e0860fcf8fc67b, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-9916-20180906-163232-5e0693ad-verify-fedora-29/log.html)

**Comment by Antique** [comment]
This should be fixed now by the latest libvirt-4.7.0-1.fc29 build.

**Comment by Gundersanne** [comment]
Thank you! 

After a fedora-29 image refresh on our side, this issue should close automatically after a while.
